### ISSUE TITLE
Can u help me?

### ISSUE THREAD

**Issue by Raxana7** [kus/csgo-modded-server#4]
![image](https://user-images.githubusercontent.com/48220802/80860245-1b917f00-8c6f-11ea-8213-f99a9daf3bc2.png)

**Comment by kus** [comment]
Are you running an internet or LAN server?

If internet, have you created and set a Steam Game Server Login Token? https://steamcommunity.com/dev/managegameservers

**Comment by Raxana7** [comment]
Are you running an internet or LAN server? internet 

If internet, have you created and set a Steam Game Server Login Token?  Yes
![image](https://user-images.githubusercontent.com/48220802/80862470-1851bf80-8c7e-11ea-8949-6c39d5e8413e.png)

**Comment by kus** [comment]
What's the rest of the error message when you expand "View problem details"?

We played on the server last night (on Linux) and no new updates since then and worked fine.

Unfortunately I won't have access to test on a Windows machine for a few days.

**Comment by kus** [comment]
Can you check your token and make sure it hasn't expired?

https://steamcommunity.com/dev/managegameservers

**Comment by Raxana7** [comment]
I checked it. It's not expired. I can open csgo server with normal csgo server.

**Comment by Raxana7** [comment]
What's the rest of the error message when you expand "View problem details"? I will check and send.

**Comment by kus** [comment]
Hey @Raxana7,

I tested it on a Windows box and got the same problem.

I updated SourceMod to latest and it works now.

To save downloading the whole CS:GO server again, move the following out of the `csgo-modded-server-master` folder:

`server/`
`steamcmd/`
`win.ini`

Delete the  `csgo-modded-server-master` folder. Go back to [here](https://github.com/kus/csgo-modded-server#running-on-windows) and download this repo, extract and copy:

`server/`
`steamcmd/`
`win.ini`

Into the `csgo-modded-server-master` folder and skip the SteamCMD step and continue on with the rest of the steps and it should work.
